pkg/build/api/v1beta3/types.go

```diff
@@ -385,6 +390,9 @@ type BuildRequest struct {
 
 	// Revision is the information from the source for a specific repo snapshot.
 	Revision *SourceRevision `json:"revision,omitempty"`
+
+	// Image is the image that triggered this build.
+	Image string `json:"image,omitempty"`
```

Comment:
I would rather see a `Trigger *BuildTriggerPolicy` field that correlates to the BuildTriggerPolicy that caused the build request (without secrets for webhooks, obviously). Builds not caused by triggers are null. That gives us a way to plumb trigger info into the build object eventually.